lib/kb/fake/bounded_context/pet_family/pet_contracts.rb

```diff
@@ -17,8 +20,23 @@ module PetContracts
           json_response 200, resource
         end
 
+        get '/v1/petcontracts/search' do
+          pet_by_chip = resource_state(:pets).detect do |pet|
+            pet['chip'] == params['chip']
+          end
+          return json_response 404, {} if pet_by_chip.nil?
```

Comment:
It's supposed to be an empty response in this case, isn't it?

Reply:
Uhm, you are right. I copied this from `contractnumber` endpoint without hesitating. 😓